### DISCUSSION TITLE
Instant Meeting을 별도 커맨드로 하면 좋을 것 같아요.

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#21] [제보하기]
가까이 갔을때 열리는 조건이 가끔 안 걸리는 경우도 있고 하네요. 
회의실 안에서는 아예 안되는거 같구요. 

그래서 별도 커맨드로 Instant Meeting이 만들어지면 좋을 것 같습니다. (게더의 버블처럼)

**Comment by shlee303** [comment]
회의실 안에서는 일부러 안열리게 정책을 정했습니다. (회의실 안에서는 회의에만 집중..!)
별도 커맨드로는, 미팅에 들어가는 상대방이 수락을 해야하는 이슈도 있어서 고민중에 있는데, 검토해보겠습니다!!!
